wire-compiler/src/main/java/com/squareup/wire/WireCompiler.kt

```diff
@@ -145,6 +149,8 @@ class WireCompiler internal constructor(
             .withAndroidAnnotations(emitAndroidAnnotations)
             .withCompact(emitCompact)
 
+        // TODO(oldergod) crashes if one tries to mix javaOut and services?
```

Comment:
what’s this?

Reply:
ahhh, no Java support for services. I think you can replace the TODO with a regular comment, “No services for Java”